challenge-22/submissions/Sylinsic/solution-template.go

```
@@ -0,0 +1,80 @@
+// If the amount cannot be made with the given denominations, return an empty map.
+func CoinCombination(amount int, denominations []int) map[int]int {
+	res := make(map[int]int)
+	if amount == 0 { 
+	    return res 
+	}
+	
+	tot := 0
+	for i := len(denominations) - 1; i >= 0; i-- {
+	    d := denominations[i]
+	    for {
+	        if amount - tot < d { 
```

golangci-lint (2.12.2)

[error] QF1006: could lift into loop condition

(staticcheck)